inc/BrainPadDisplay.h

```diff
@@ -0,0 +1,25 @@
+#ifndef BPDISPLAY_H
+#define BPDISPLAY_H
+
+#include "Image.h"
+#include "MbedI2C.h"
+#include "stdint.h"
+
+namespace codal {
+   class BrainPadDisplay {  
+  
+		uint8_t data[2];
+		int DeviceAddress = 0x78;
+		uint8_t byte;
+
+		public:
+		codal::_mbed::I2C i2c;
```

Comment:
Does this need to be public?

Reply:
I'll make it private and codal and see if it works.